Documentation/ProjectFile/prj/processes/process/THERMAL_TWOPHASE_FLOW_PP/material_property/t_specific_heat_capacity_air.md

```diff
@@ -0,0 +1 @@
+Specific heat capacity of the dry air.
```

Comment:
Here I use air only because in the benchmark test, air is presented.
but of course, this "air" can be further replaced by other gas component like hydrogen, oxygen, or nitrogen.